Makefile

```diff
@@ -60,66 +60,19 @@ test-e2e: export OASIS_INDEXER_E2E = true
 test-e2e:
 	@$(GO) test -race -coverpkg=./... -coverprofile=coverage.txt -covermode=atomic -v ./tests/e2e
 
-fill-cache-for-e2e-regression:
-	@$(MAKE) fill-cache-for-e2e-regression-eden
-	@$(MAKE) fill-cache-for-e2e-regression-damask
+e2e_regression_suites := eden damask
 
-fill-cache-for-e2e-regression-eden: nexus
-	@./tests/e2e_regression/ensure_consistent_config.sh
-	./nexus --config tests/e2e_regression/eden/e2e_config_1.yml analyze
-	./nexus --config tests/e2e_regression/eden/e2e_config_2.yml analyze
-
-fill-cache-for-e2e-regression-damask: nexus
-	@./tests/e2e_regression/ensure_consistent_config.sh
-	./nexus --config tests/e2e_regression/damask/e2e_config_1.yml analyze
-	./nexus --config tests/e2e_regression/damask/e2e_config_2.yml analyze
+ensure-consistent-config-for-e2e-regression:
+	for suite in $(e2e_regression_suites); do ./tests/e2e_regression/ensure_consistent_config.sh $$suite; done
 
 # Run the api tests locally, assuming the environment is set up with an oasis-node that is
 # accessible as specified in the config file.
-test-e2e-regression:
-	@$(MAKE) test-e2e-regression-eden
-	@$(MAKE) test-e2e-regression-damask
-
-test-e2e-regression-eden: nexus
-	@./tests/e2e_regression/ensure_consistent_config.sh
-	./nexus --config tests/e2e_regression/eden/e2e_config_1.yml analyze
-	./nexus --config tests/e2e_regression/eden/e2e_config_2.yml analyze
-	@$(ECHO) "$(CYAN)*** Analyzers finished; starting api tests...$(OFF)"
-	./tests/e2e_regression/run.sh eden
-
-
-test-e2e-regression-damask: nexus
```

Comment:
One of the reasons we had these as separate targets was that we want to run them as separate (and parallel!) CI jobs.
Example:

Your call on what we do. Ideas:
- we keep it simple and just call sth like `make nexus ensure-consistent-config-for-e2e-regression; ./tests/e2e_regression/run.sh eden`
- we parametrize this make target:

   ```Makefile
   test-e2e-regression: nexus ensure-consistent-config-for-e2e-regression
   ifndef SUITE
   	for suite in $(e2e_regression_suites); do ./tests/e2e_regression/run.sh $$suite; done
   else
   	./tests/e2e_regression/run.sh $(SUITE)
   endif
   ```
   and then call it from CI as `SUITE=eden make test-e2e-regression`.

Reply:
ah yeah ok. or maybe they could set e2e_regression_suites outside

Reply:
cool, very elegant :+1: